source/_docs/installation/freenas.markdown

```diff
@@ -33,7 +33,7 @@ Create an `/etc/rc.local` file to enable Home Assistant to start when the jail s
 
 
 ```bash
-# cd / && mkdir /home && /home/.homeassistant/
+# cd / && mkdir /home && mkdir /home/.homeassistant/
```

Comment:
Why not simply use: `mkdir -p /home/.homeassistant` ?

Reply:
I'm sure that would also work.  I was just proposing a simple tweak to make the command accurate.